src/pages/[platform]/deploy-and-host/fullstack-branching/mono-and-multi-repos/index.mdx

```diff
@@ -43,19 +43,18 @@ You might have different frontend and backend teams that maintain their own repo
 ![Screenshot of the App Settings section in the Amplify console with Amplify Deploy listed under Auto-detected frameworks](/images/gen2/fullstack-branching/multirepo1.png)
 
 4. Once you choose **Save and deploy**, your backend project will build.
-
 ## Deploy the frontend app
 
 1. Now let's set up the frontend app and connect to the deployed backend.
 
-```
+```bash showLineNumbers={false}
 npm create next-app@14 -- multi-repo-example --typescript --eslint --no-app --no-src-dir --no-tailwind --import-alias '@/*'
 
 ```
 
 2. Install Amplify dependencies.
 
-```
+```bash showLineNumbers={false}
```

Comment:
````suggestion
```bash title="Terminal" showLineNumbers={false}
````

Reply:
can we add this for every bash example?

Reply:
All done.